clientlibs/js/src/UpgradeClient.ts

```diff
@@ -108,15 +109,15 @@ export default class UpgradeClient {
     this.token = options?.token;
     this.clientSessionId = options?.clientSessionId || uuid.v4();
     this.api = {
-      init: `${hostUrl}/api/v4/init`,
-      getAllExperimentConditions: `${hostUrl}/api/v4/assign`,
-      markExperimentPoint: `${hostUrl}/api/v4/mark`,
-      setGroupMemberShip: `${hostUrl}/api/v4/groupmembership`,
-      setWorkingGroup: `${hostUrl}/api/v4/workinggroup`,
-      failedExperimentPoint: `${hostUrl}/api/v4/failed`,
-      getAllFeatureFlag: `${hostUrl}/api/v4/featureflag`,
-      log: `${hostUrl}/api/v4/log`,
-      logCaliper: `${hostUrl}/api/v4/log/caliper`,
+      init: `${hostUrl}/api/v5/init`,
+      getAllExperimentConditions: `${hostUrl}/api/v5/assign`,
+      markExperimentPoint: `${hostUrl}/api/v5/mark`,
+      setGroupMemberShip: `${hostUrl}/api/v5/groupmembership`,
+      setWorkingGroup: `${hostUrl}/api/v5/workinggroup`,
+      failedExperimentPoint: `${hostUrl}/api/v5/failed`,
+      getAllFeatureFlag: `${hostUrl}/api/v5/featureflag`,
+      log: `${hostUrl}/api/v5/log`,
+      logCaliper: `${hostUrl}/api/v4/logCaliper`,
```

Comment:
Should these be:

logCaliper: `${hostUrl}/api/v5/log/caliper`,
altUserIds: `${hostUrl}/api/v5/useraliases`,
addMetrics: `${hostUrl}/api/v5/metric`,

Reply:
Updated this code